transports/webrtc/src/tokio/upgrade.rs

```diff
@@ -25,6 +25,7 @@ use futures::future::Either;
 use futures_timer::Delay;
 use libp2p_identity as identity;
 use libp2p_identity::PeerId;
+use log::debug;
```

Comment:
I'd prefer to not import `log` macros. I find `log::` to be quite descriptive and refactoring code often results in a lot of churn on which ones of these one needs to import / remove from the imports.

Reply:
Ok

Reply:
Done